### ISSUE TITLE
Particles are not drawn correctly!

### ISSUE THREAD

**Issue by wmaestrots** [opentibiabr/otclient#230]
### Steps to reproduce (include any configuration/script required to reproduce)
- Download my module (**game_testparticles**) and simple test it
- Link for game_testparticles download : [game_testparticles.rar](https://xfl.jp/5e2uLl) [**website : FileSend.jp , free hosting**]
- (just copy and paste it into /modules/ and add it ofcourse in game_interface like other modules to load it).
- On Mehah's client particles **aren't drawn** **correctly**.

### Expected behaviour
Should work like on .gif recorded from Edubart's clean client.
![](https://i.imgur.com/VHeUVNV.gif)

**Same module in Mehah's Client (don't mind my custom stuff, just look what happen if you click same button):**
![](https://i.gyazo.com/e03bd99016f5808b3f41a0f941ffb0af.gif)

### Actual behaviour
Drawing particles is screwed up, because Mehah made his optimalization, drawPools and other tricky-trick's, that's so far I saw in sources, he forget to update Particles algorithm/logic, so it's just better if creator himself - Mehah - will fix it properly.

tl;dr : Client goes freeze and not responding && particles algorithm/logic is outdated/not compatible with new Mehah's draw techniques.
